providers/provider-pojo/src/main/java/org/apache/servicecomb/provider/pojo/Invoker.java

```diff
@@ -57,7 +58,10 @@ public InvokerMeta(ReferenceConfig referenceConfig,
 
     // initialized and meta not changed
     public boolean isValid() {
-      return swaggerConsumer != null && microserviceMeta == referenceConfig.getMicroserviceMeta();
+      SCBStatus currentStatus = SCBEngine.getInstance().getStatus();
+      return SCBStatus.UP.equals(currentStatus)
```

Comment:
when shutdown, this will cause createInvokerMeta again, not throw exception

Reply:
Done